### PR TITLE
feat: upgrade zone.js to v0.7.2

### PR DESCRIPTION
**Pull request by mhevery**

### CODE REVIEW

File: modules/@angular/core/test/fake_async_spec.ts [hunk 1]
```diff
@@ -95,7 +95,7 @@ export function main() {
             resolvedPromise.then((_) => { throw new Error('async'); });
             flushMicrotasks();
           })();
-        }).toThrowError('Uncaught (in promise): Error: async');
+        }).toThrowError(/Uncaught \(in promise\)\: Error\: async/);
```
Comment (e-cloud) [inline modules/@angular/core/test/fake_async_spec.ts]: is it necessary to escape colon?

**Comment by alanpurple** [comment]
will this be 2.3.0? or 2.4.0?

File: modules/@angular/core/test/fake_async_spec.ts (hunk 1 above)
Comment (IgorMinar) [inline modules/@angular/core/test/fake_async_spec.ts]: no, it's not. @mhevery please fix.

File: npm-shrinkwrap.clean.json [hunk 2]
```diff
@@ -2509,7 +2526,10 @@
       "version": "5.0.0",
       "dependencies": {
         "glob": {
-          "version": "7.1.1"
+          "version": "7.0.6"
```
Comment (IgorMinar) [inline npm-shrinkwrap.clean.json]: glob is getting downgraded here? that makes no sense.

Please either manually remove the diff from this PR other than zone.js changes, or we can investigate together what's going on with npm - all these version changes should not be here.

**Review by IgorMinar — DISMISSED** [review]
needs cleanup

File: modules/@angular/core/test/fake_async_spec.ts (hunk 1 above)
Comment (mhevery) [inline modules/@angular/core/test/fake_async_spec.ts]: fixed

File: npm-shrinkwrap.clean.json (hunk 2 above)
Comment (mhevery) [inline npm-shrinkwrap.clean.json]: Removed all of these changes.

**Review by mhevery — COMMENTED** [review]
cleaned up

**Comment by vsavkin** [comment]
Merged https://github.com/angular/angular/commit/c4bbafc2911b7514945ba461d9a32b9179da0d54

**Pull request closed without merging**